packages/sit-onyx/src/components/OnyxPageLayout/OnyxPageLayout.stories.ts

```diff
@@ -93,3 +102,24 @@ export const WithPartialFooter = {
     footerAsideSidebar: true,
   },
 } satisfies Story;
+
+/** A standard page in skeleton mode. */
+export const WithSkeletonComponents = {
+  args: {
+    skeleton: true,
+    default: () => [
+      h(OnyxRadioGroup, {
+        label: "Color choices",
+        options: EXAMPLE_OPTIONS,
+        modelValue: EXAMPLE_OPTIONS[0].value,
+      }),
+      h(
+        "div",
+        {
+          style: { display: "flex", gap: "0.5rem", paddingLeft: "0.5rem" },
+        },
+        [h(OnyxButton, { label: "Reset", mode: "outline" }), h(OnyxButton, { label: "Submit" })],
```

Comment:
If the buttons are labeled like this, I'd prefer to wrap it in a form/OnyxForm so its a "correct" form example :) And maybe add a little spacing between the radio group and the buttons

Reply:
We have an issue there. The provided skeleton prop doesn't work when we have an OnyxForm wrapped in OnyxPageLayout. There is a ticket for that -> https://github.com/SchwarzIT/onyx/issues/1955

Reply:
Ah okay, then maybe use a regular `<form>` instead :) Imo we should not promote creating forms without any kind of form component since validation etc. does not work then